### ISSUE TITLE
Show model performance on validation set

### ISSUE THREAD

**Issue by tobiasvanderwerff** [tobiasvanderwerff/RNN-for-Joint-NLU#1]
Currently, we can only see performnace on the training set when training the model. In the training loop, add code that runs the model on a validation set and shows the performance.
